engines/private/private.h

```diff
@@ -0,0 +1,300 @@
+#ifndef PRIVATE_H
+#define PRIVATE_H
+
+#include "common/random.h"
+#include "common/serializer.h"
+#include "engines/advancedDetector.h"
+#include "engines/engine.h"
+#include "gui/debugger.h"
```

Comment:
You do not have the debug `Console` implemented, so it's better to remove all of it.

Reply:
I'm not sure what it could be removed, since the `Console` class is used.

Reply:
It is empty. So, remove the whole `Console` class too.

Reply:
Done.